v2/internal/testcommon/kube_per_test_context.go

```diff
@@ -157,7 +159,6 @@ func (ctx KubeGlobalContext) ForTestWithConfig(t *testing.T, cfg config.Values)
 		Verify:              verify,
 		Match:               match,
 		scheme:              scheme,
-		Ctx:                 context,
```

Comment:
Why remove context here?

Reply:
I pulled it up into the `PerTestContext` rather than the `KubePerTestContext` so that I could use it in the semaphore acquire call.